csharp/src/Glacier2/SessionHelper.cs

```diff
@@ -462,33 +458,13 @@ private void
         private void
         DispatchCallback(Action callback, Connection? conn)
         {
-            if (_dispatcher != null)
-            {
-                _dispatcher(callback, conn);
-            }
-            else
-            {
-                callback();
-            }
+            callback();
```

Comment:
If we are keeping this for when we add a replacement, we can add a TODO comment here, otherwise remove it.

Reply:
Removed, thanks.